docs/installation.adoc

```diff
@@ -41,7 +41,7 @@ Configured using `ENABLE_LOCAL_MODE`. Set it to`True` for _Local Mode_ and `Fals
 ==== Public Mode
 
 - To use when deploying the app on public server, where different people can access the app.
-- No default account is created.
+- Also creates an Admin User, with credentials from secret config (instance/config.py)
```

Comment:
This seems redundant. It's not needed in description of any of the modes, but needs to be mentioned separately.